README.md

```diff
@@ -1,10 +1,13 @@
 # READ THIS FIRST!
 
-SvelteKit is still in beta. Expect bugs! Read more [here](https://svelte.dev/blog/sveltekit-beta), and track progress towards 1.0 [here](https://github.com/sveltejs/kit/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.0).
+SvelteKit is still in beta. Expect bugs! Read more
+[here](https://svelte.dev/blog/sveltekit-beta), and track progress towards 1.0
+[here](https://github.com/sveltejs/kit/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.0).
 
 ## Documentation
 
-Please see [the documentation](https://kit.svelte.dev/docs) for information about getting started and developing with SvelteKit.
+Please see [the documentation](https://kit.svelte.dev/docs) for information
```

Comment:
can you revert the formatting changes? manual line breaks are a pain. if i add a word i need to update every line following. text editors have word wrap to make lines shorter when reading, so there's no need to manually manage it

Reply:
I use zero automation tools. Anything that happened here is a direct result of the command scripts or the stuff that Svelte kit automatically installed into my VS code. Hugely against all of this stuff